src/WordsFunctionArgException.java

```diff
@@ -1,26 +1,20 @@
 
-public class WordsFunctionArgException extends Exception {
+@SuppressWarnings("serial")
+public class WordsFunctionArgException extends WordsException {
```

Comment:
I'm leaving that up to the programmer using the exception in this case. `receivedArg` and `expectedArg` are both String fields, so you can put in your list of arguments, and list of what you received. Think of it somewhat like the generic "usage" statement that most C programs have when they are run with the wrong number of arguments. You can say what you expected (the usage), and what you received (the type of each parameter).